dlog/plonk/src/range.rs

```diff
@@ -0,0 +1,7 @@
+use std::ops::Range;
+
+pub const PERM:     Range<usize> = 0..2;
```

Comment:
Why do you have them using non-overlapping ranges? Isn't it ok for them to use the same alphas?

Reply:
see the previous comment